charts/athens-proxy/templates/jaeger-deploy.yaml

```diff
@@ -4,10 +4,7 @@ kind: Deployment
 metadata:
   name: {{ template "fullname" . }}-jaeger
   labels:
-    app: {{ template "fullname" . }}-jaeger
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-    release: "{{ .Release.Name }}"
-    heritage: "{{ .Release.Service }}"
+    {{- include "athens.metaLabels" (dict "appSuffix" "-jaeger" "Values" .Values "Release" .Release "Chart" .Chart) | nindent 4 }}
```

Comment:
This feels a bit gross since it has to be aware of the values being used by the underlying helper, but I wanted to find a way to re-use the helper for these couple of unique cases that have a different name specified currently. 

FWIW, these differences seem a bit strange, I'd normally expect all of the resources created by the helm chart to have this same app name. However, I tried to keep things as passive as possible.

I did consider creating a separate helper function that was `metaLabelsWithoutAppName` that would be re-used by the new helper and could be used here and have the `app` and `app.kubernetes.io/name` be set here explicity. I favored this approach but am happy to change things up

Reply:
I agree, not the nicest include I've seen, but it helps keeping the labels consistent and may not be worth to add an extra helper.